skcosmo/feature_selection/_base.py

```diff
@@ -0,0 +1,49 @@
+"""
+Sequential feature selection
+"""
+
+from .._selection import (
+    _CUR,
+    _FPS,
+    _PCovCUR,
+    _PCovFPS,
+)
+
+
+class FPS(_FPS):
+    """Transformer that performs Greedy Feature Selection using Farthest Point Sampling."""
+
+    def __init__(self, **kwargs):
+        super().__init__(selection_type="feature", **kwargs)
```

Comment:
So if I understand correctly, the only real difference between the FPS from `feature_selection/_base.py` and `sample_selection/_base.py` is that in feature selection the `FPS` class (and `CUR`, etc.) are is initialized with `selection_type='feature'` and the sample selectors are initialized with `selection_type='sample'`. In principle this is just something the user could choose themselves -- is the reason for keeping two different modules for "future compatibility" where sample and feature selection might diverge with potential future changes/improvements?

Reply:
Exactly! We may be able to take advantage of some speedups with feature selection that we can't with sample selection, or vice versa. If this becomes moot, it'll be an easy fix, whereas vice-versa may not be easily possible.